tests/functional/includes/date.mock.js

```diff
@@ -0,0 +1,14 @@
+// Highjack the Date object so that our tests don't suddenly start failing...
```

Comment:
Unfortunately, that will not work. Some examples I wanted to test, including the splash example on the home page, use `Date.now()`.